src/StdChains.sol

```diff
@@ -88,7 +96,7 @@ abstract contract StdChains {
     }
 
     // set chain info, with priority to argument's rpcUrl field.
-    function setChain(string memory chainAlias, Chain memory chain) internal virtual {
+    function setChain(string memory chainAlias, ChainData memory chain) internal virtual {
         require(
             bytes(chainAlias).length != 0, "StdChains setChain(string,Chain): Chain alias cannot be the empty string."
```

Comment:
Can you update these error messages to say `ChainData`, and the description on top of the file?

Also, do you think it's useful, to be able to "convert" `Chain` into `ChainData` so you can pass it to `setChain`, by adding an alternative signature:

```
chain = getChain(...)
chain.name = ...
setChain(..., chain)
```

Otherwise, LGTM, @mds1 can merge when ready.

Reply:
Done. https://github.com/foundry-rs/forge-std/pull/268/commits/43ff37017acaa235d5c99d42b73b7fa665a530df